platform/linux-generic/m4/odp_dpdk.m4

```diff
@@ -5,9 +5,21 @@ pktio_dpdk_support=no
 AC_ARG_WITH([dpdk-path],
 [AS_HELP_STRING([--with-dpdk-path=DIR], [path to dpdk build directory])],
     [DPDK_PATH="$withval"
+     pktio_dpdk_support=yes],[])
+
+if test "x$DPDK_PATH" = "xsystem" ; then
```

Comment:
If I understood this correctly, a user would have to use '--with-dpdk-path=system' configure option for this to work. Wouldn't a more standard way of doing this be to add e.g. '--enable-dpdk-support' flag in the same manner as with netmap pktio.

Reply:
Good question. For more or less typical case (non-system DPDK) this would require adding two flags (`--enable-dpdk-support --with-dpdk-path=PATH_TO_DPDK`).

Reply:
I would suggest going forward with this approach (two separate flags).

If the user only gives '--with-dpdk-path= PATH_TO_DPDK' flag, you can presumably handle this in the configure script (so that only one flag has to be used).